docs/conf.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+#
+# lstchain documentation build configuration file, created by
+# sphinx-quickstart on Tue Aug 11 14:07:30 2020.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+ needs_sphinx = '1.5'
```

Comment:
remove whitespace at the beginning of the line

Reply:
already done in https://github.com/cta-observatory/cta-lstchain/pull/761/commits/23b5bc91bbf0088824dc7cdfa507084f8583fa4f